modules/installation-configuration-parameters.adoc

```diff
@@ -2192,7 +2192,7 @@ When running your cluster on GCP 64-bit ARM infrastructures, ensure that you use
       osDisk:
         diskType:
 |The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type].
-|The default disk type for all machines. Control plane nodes must use the `pd-ssd` disk type. Compute nodes can use the `pd-ssd`, `pd-balanced`, or `pd-standard` disk types.
+|The default disk type for all machines. Valid values are `pd-balanced`, `pd-ssd`, `pd-standard`, `hyperdisk-balanced`. Control plane machines cannot use the `pd-standard` disk type, so if you specify it here, you must specify a valid disk type using the `controlPlane.platform.gcp.osDisk.diskType` parameter. The default value is `pd-ssd`.
```

Comment:
```suggestion
|The default disk type for all machines. Valid values are `pd-balanced`, `pd-ssd`, `pd-standard`, or `hyperdisk-balanced`. The default value is `pd-ssd`. If you specify a value here, you must specify a valid disk type using the `controlPlane.platform.gcp.osDisk.diskType` parameters. Control plane machines cannot use the `pd-standard` disk type. 
```
I re-wrote the final sentences based on what I think you are trying to say. If it doesn't work, disregard my edits.

Reply:
Thanks Janelle, I will reword slightly with your suggestion in mind.